src/pages/Lend/LendPage.vue

```diff
@@ -25,55 +32,126 @@
 <script>
 import WwwPage from '@/components/WwwFrame/WwwPage';
 import GridLoanCard from '@/components/LoanCards/GridLoanCard';
+import KvLoadingSpinner from '@/components/Kv/KvLoadingSpinner';
+import KvPagination from '@/components/Kv/KvPagination';
 import loanCardQuery from '@/graphql/query/loanCardData.graphql';
 import _get from 'lodash/get';
+import _invokeMap from 'lodash/invokeMap';
 import _map from 'lodash/map';
+// import _mapValues from 'lodash/mapValues';
+import _merge from 'lodash/merge';
+import numeral from 'numeral';
+
+const loansPerPage = 12;
+
+const urlParamTransform = {
+	page: {
+		to({ offset }) {
+			const page = Math.floor(offset / loansPerPage) + 1;
+			return page > 1 ? page : null;
+		},
+		from({ page }) {
+			const pagenum = numeral(page).value() - 1;
+			return { offset: pagenum > 0 ? loansPerPage * pagenum : 0 };
+		}
+	},
+};
```

Comment:
Would love to walk through this portion of the code with you at some point @emuvente